pkg/skaffold/runner/listen.go

```diff
@@ -54,6 +54,7 @@ func (l *SkaffoldListener) WatchForChanges(ctx context.Context, out io.Writer, d
 	}
 
 	// exit if file monitor fails the first time
+	preRun()
```

Comment:
this is just for invalidating the cache right? can we call it `invalidateDependencyCache` or something more descriptive?